workspace/src/localization/localization_py/package.xml

```diff
@@ -14,7 +14,7 @@
 
   <exec_depend>python3-matplotlib</exec_depend>
   <exec_depend>python3-numpy</exec_depend>
-  <exec_depend>python3-scipy</exec_depend>
+  <exec_depend>python3-scipy="1.8"</exec_depend>
```

Comment:
nice! Is that allowed? Does that work?

Reply:
@AaronYoung5 yeah seems to work for me. https://stackoverflow.com/questions/75693898/how-to-set-a-specific-version-of-exec-depend-in-ros-package-xml

Reply:
I think it should be `<exec_depend version_gte="1.8">python3-scipy</exec_depend>` then, right?

Reply:
🤷 worked either way for me... implemented it the way you said